src/pages/home/base/mobile-core/identity/tutorials/visitor-tracking.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Visitor tracking between an app and the mobile web
+description: Learn how to make add visitor tracking between an app and the mobile web.
+keywords:
+- Identity for Mobile Core
+- Identity
+- Mobile Core
+- Product overview
+---
+
+import Tabs from '../tabs/visitor-tracking.md'
+
+# Visitor tracking between an app and the mobile web
+
+If your app opens mobile web content, you need to ensure that visitors are not identified separately as they move between the native and mobile web.
+
+## Visitor IDs in apps
+
+The Mobile SDK generates a unique visitor ID when the app is installed. This ECID is stored in persistent memory on the mobile device and is sent with every hit. The ECID is removed when the user uninstalls the app or when the user sets the Mobile SDK global privacy status to `optedout`.
+
+<InlineAlert variant="info" slots="text"/>
+
+When the Mobile SDK privacy status is set to `optedout`, and the ECID is removed, a new unique visitor ID (ECID) is generated when the user sets the global privacy status to `optedin`.
+
+<InlineAlert variant="info" slots="text"/>
+
+App visitor IDs persist through upgrades.
+
+## Visitor IDs in the mobile web
+
+Typical mobile web implementations use the same standard analytics `s_code.js` or `AppMeasurement.js` that is used in desktop sites. The JavaScript libraries have their own methods of generating unique visitor IDs, which causes a different visitor ID to be generated when you open mobile web content from your app.
```

Comment:
Typical mobile web implementations use one of the Adobe JavaScript libraries available for web platform, such as Adobe Experience Platform Web SDK, or `AppMeasurement.js`. These libraries have their own ...